src/health/probe_test.go

```diff
@@ -71,7 +71,7 @@ func TestProbe_validateAndSetDefaults(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			p := &Probe{
 				Exec:             tt.fields.Exec,
-				HttpGet:          tt.fields.HttpGet,
+				Http:             tt.fields.Http,
```

Comment:
Can you please add a test to validate that the `Method` and the `StatusCode` are set correctly?

Reply:
⚡